concurrent_test.go

```diff
@@ -24,6 +24,22 @@ import (
 	"go.etcd.io/bbolt/internal/common"
 )
 
+type interval struct {
+	start time.Duration
+	end   time.Duration
```

Comment:
`start` and `end` look like timestamps, but they are min/max durations. So I'd like to change it to 

```
type duration struct {
    min time.Duration
    max time.Duration
}
```

Reply:
updated.